packages/grafana-ui/src/components/Forms/Checkbox.tsx

```diff
@@ -44,34 +44,20 @@ export const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
 
 export const getCheckboxStyles = stylesFactory((theme: GrafanaTheme2) => {
   const labelStyles = getLabelStyles(theme);
-  const checkboxSize = '16px';
+  const checkboxSize = 2;
+  const labelPadding = 1;
+
   return {
-    label: cx(
-      labelStyles.label,
-      css`
-        padding-left: ${theme.spacing(1)};
-        white-space: nowrap;
-        cursor: pointer;
-      `
-    ),
-    description: cx(
-      labelStyles.description,
-      css`
-        line-height: ${theme.typography.bodySmall.lineHeight};
-        padding-left: ${theme.spacing(1)};
-      `
-    ),
     wrapper: css`
       position: relative;
-      padding-left: ${checkboxSize};
       vertical-align: middle;
-      min-height: ${theme.spacing(3)};
+      font-size: 0;
     `,
     input: css`
       position: absolute;
       top: 0;
       left: 0;
-      width: 100%;
+      width: 100% !important; // global styles unset this
       height: 100%;
       opacity: 0;
```

Comment:
```suggestion
      opacity: 0;
      cursor: pointer;
```
I think we need to add pointer cursor to the input directly, otherwise it shows the default cursor on hover.

Reply:
Checkboxes do not normally have the pointer cursor, but it looks like they did in Grafana previously. We want to keep that I presume?

Reply:
I think it'd be nice to keep them, this also enables cursor pointer when hovering on label, which clearly indicates that it's clickable.

Reply:
Resolved this in one way or another

Reply:
hmm, but now the label doesn't have `cursor: pointer` even though it's clickable.